index.js

```diff
@@ -18,7 +18,10 @@ function npmInstall(packageName, callback) {
   npm.load({}, function (error, npm) {
     npm.prefix = tmpDir;
     if (packageName) {
-      npm.commands.install([packageName], function () {
+      npm.commands.install([packageName], function (err, deps) {
+        var location = deps[deps.length - 1][1]
```

Comment:
It gives you the relative path of the module, e.g.  "../jsize/tmp/node_modules/module"
That way you can construct the full absolute path to the module: "/Users/elfet/jsize/tmp/node_modules/module/package.json"
which means you can now pass relative paths to jsize!, e.g. jsize ../my-module